tests/memoryview/numpy_memoryview.pyx

```diff
@@ -734,3 +734,18 @@ def same_type_after_arrays_composite():
     cdef SameTypeAfterArraysStructComposite element
     arr = np.ones(2, np.asarray(<SameTypeAfterArraysStructComposite[:1]>&element).dtype)
     cdef SameTypeAfterArraysStructComposite[:] memview = arr
+
+
+ctypedef fused np_numeric_t:
+    np.float64_t
+
+def test_issue5401(np_numeric_t[:] A):
```

Comment:
I suspect @scoder is likely to complain about the non-descriptive naming here.

I'd suggest `test_invalid_buffer_fused_memoryview` maybe?

Reply:
done